src/e2e/puppeteer/helpers/getEditingText.ts

```diff
@@ -5,7 +5,7 @@ import { Page } from 'puppeteer'
  */
 const getEditingText = (page: Page) =>
   page.evaluate(() => {
-    return document.querySelector('.editing .editable')?.innerHTML
+    return document.querySelector('[data-editing=true] .editable')?.innerHTML
```

Comment:
`.editable` should probably be pulled dynamically from the `editable` recipe rather than hardcoded here.

Reply:
This was updated in https://github.com/cybersemics/em/blob/51c8fa94f41ef99c466a9e659ce216a4b38913fd/src/e2e/iOS/helpers/getEditingText.ts but not https://github.com/cybersemics/em/blob/51c8fa94f41ef99c466a9e659ce216a4b38913fd/src/e2e/puppeteer/helpers/getEditingText.ts.

Like in the iOS helper, I'm curious if a PandaCSS will even work inside the puppeteer browser page environment. It's worth a try.